macro/run_simScript.py

```diff
@@ -94,6 +94,9 @@
 parser.add_argument("--phiRandom", dest="phiRandom",  help="only relevant for muon background generator, random phi", required=False, action="store_true")
 parser.add_argument("--Cosmics",   dest="cosmics",  help="Use cosmic generator, argument switch for cosmic generator 0 or 1", required=False,  default=None)
 parser.add_argument("--MuDIS",     dest="mudis",  help="Use muon deep inelastic scattering generator", required=False, action="store_true")
+parser.add_argument("--xRange", dest="mudis_x_range", nargs=2, help="lower and upper x limit of DIS interaction", required=False, default=[-999,999], type=float)
+parser.add_argument("--yRange", dest="mudis_y_range", nargs=2, help="lower and upper y limit of DIS interaction", required=False, default=[-999,999], type=float)
+parser.add_argument("--zRange", dest="mudis_z_range", nargs=2, help="lower and upper z limit of DIS interaction", required=False, default=[-999,999], type=float)
```

Comment:
Might be worth having a look whether any other generators use a set range, in which case we can also use it there.
I don't like the default values, they should be something reasonable-ish.

The lines are too long.

The units expected should be mentioned.